pages/index.js

```diff
@@ -9,6 +9,7 @@ import { Button } from "../components/Button";
 
 import { vocs } from "../lib/db";
 import Form from "../components/Form";
+import NavigationBar from "../components/Navigation/NavigationBar";
```

Comment:
What for?

Reply:
Leftover from the development process. :)